src/PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.cpp

```diff
@@ -346,31 +346,47 @@ void spacetime_normal_vector(
 }
 
 template <size_t SpatialDim, typename Frame, typename DataType>
-tnsr::ii<DataType, SpatialDim, Frame> extrinsic_curvature(
+void extrinsic_curvature(
+    const gsl::not_null<tnsr::ii<DataType, SpatialDim, Frame>*> ex_curvature,
     const Scalar<DataType>& lapse,
     const tnsr::I<DataType, SpatialDim, Frame>& shift,
     const tnsr::iJ<DataType, SpatialDim, Frame>& deriv_shift,
     const tnsr::ii<DataType, SpatialDim, Frame>& spatial_metric,
     const tnsr::ii<DataType, SpatialDim, Frame>& dt_spatial_metric,
     const tnsr::ijj<DataType, SpatialDim, Frame>&
         deriv_spatial_metric) noexcept {
-  const DataType half_over_lapse = 0.5 / get(lapse);
+  destructive_resize_components(ex_curvature, get_size(get(lapse)));
+  for (auto& component : *ex_curvature) {
```

Comment:
Do we need to initialize this to zero, if we reorder the operations below to first set the `ex_curvature` to `-dt_spatial_metric`?